charts/deployment-operator/values.yaml

```diff
@@ -114,7 +114,11 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
-resources: {}
+resources:
+  requests:
+    ephemeral-storage: "4Gi"
```

Comment:
we should be setting cpu/memory requests too (that's in another ticket).

Reply:
(if this is the root cause, I think we should set 10Gi as our limit just to be safe though)